components/NewNFTBook.vue

```diff
@@ -430,6 +457,8 @@ const { getStripeConnectStatusByWallet } = storeToRefs(stripeStore)
 const { token } = storeToRefs(bookStoreApiStore)
 const nftStore = useNftStore()
 
+const UPLOAD_FILESIZE_MAX = 20 * 1024 * 1024
```

Comment:
```suggestion
const UPLOAD_FILESIZE_MAX = 1 * 1024 * 1024
```

Consider integrating sharp if we really think user cannot resize image properly